src/App.js

```diff
@@ -1,14 +1,30 @@
+/* External imports */
+import { Routes, Route } from 'react-router-dom';
+
+/* Style imports */
 import "./styles/styles.css";
-import { Navbar } from './components/Navbar';
 
+/* Component imports */
+import { Navbar } from './components/Navbar';
 import { Footer } from './components/Footer';
-import { LandingPage } from "./pages/LandingPage";
+
+/* Page imports */
+import { LandingPage, Products, Cart, WishList, Signup, Login, Logout } from './pages/';
+
 
 function App() {
     return (
         <div className="App">
             <Navbar  />
-            <LandingPage  />
+                <Routes>
+                    <Route path="/products" element={<Products  />} />
```

Comment:
You can use nested routing here as <Header/> will remain same on every route.
```
  <Route path="/" element={<App />}>
        <Route index element={<Home />} />
        <Route path="products" element={<ProductsListing />} />
        <Route path="/products/:id" element={<SingleProductPage />} />
        <Route path="cart" element={<Cart />} />
        <Route path="address-management" element={<AddressManagement />} />
        <Route path="wishlist" element={<WishList />} />
  </Route>
```
And use <Outlet /> for rendering route specific components.